sample-functions/AlpineFunction/Dockerfile.armhf

```diff
@@ -1,8 +1,9 @@
-FROM armhf/alpine:3.5
+FROM arm32v6/alpine:latest
```

Comment:
Can we keep the convention of using a specific tag for Alpine Linux? Upgrading 3.5 -> 3.6 is fine with me, but we should pin that.